llvm/test/tools/llvm-objcopy/MachO/globalize-symbol.test

```diff
@@ -0,0 +1,150 @@
+# RUN: yaml2obj %s -o %t
+# RUN: llvm-objcopy --wildcard --globalize-symbol="*" %t %t.copy
+# RUN: llvm-readobj --symbols %t.copy | FileCheck %s
+
+# CHECK: Symbols [
```

Comment:
Nit: when we have a big block like this of CHECK followed by lots of CHECK-NEXT, we tend to add a few spaces after the colon before the actual CHECK, per the suggestion, to make the checked-for text line up with the remaining lines below. Applies throughout.
```suggestion
# CHECK:      Symbols [
```